src/components/headerComponent.vue

```diff
@@ -228,16 +228,6 @@ export default defineComponent({
       toast.add({severity:'success', summary: t('home.switchDefault') , detail: data.name, group: 'br-custom', life: 3000});
     }
 
-    emitter.on('TRIGGER_SWITCH_DEFAULT_ACCOUNT_MODAL', (payload) => {
-      openSetDefaultModal.value = payload;
-    });
-
-    emitter.on('CLOSE_SET_DEFAULT_ACCOUNT_MODAL', payload => {
-      if(payload){
-        openSetDefaultModal.value = false;
-      }
-    });
-
```

Comment:
is SetAccountDefaultModal.vue still a relevant? if not, please remove it

Reply:
also remove the variable 'openSetDefaultModal'

Reply:
noted, just revised and remove SetAccountDefaultModal.vue